docs/pages/get-started/introduction.mdx

```diff
@@ -0,0 +1,27 @@
+---
+title: Introduction
+description: Get started creating apps with Expo.
+hideTOC: true
+---
+
+import { BoxLink } from '~/ui/components/BoxLink';
+import { CALLOUT } from '~/ui/components/Text';
+import { BookOpen02Icon, Earth01Icon } from '@expo/styleguide-icons';
+import { HelloWave } from '~/scenes/get-started/introduction/HelloWave';
+
+## Welcome! <HelloWave />
+
+Expo is framework that makes developing Android and iOS apps easier. Our framework provides file-based routing, a standard library of native modules, and much more. Expo is open source with an active community on [GitHub](https://github.com/expo/expo) and [Discord](https://chat.expo.dev).
+
+We also make [Expo Application Services (EAS)](https://expo.dev/eas), a set of services that complement the Expo framework in each step of the development process.
+
+{/* There are two ways to get started making apps with Expo: */}
+
+{/* <BoxLink title="Automated" description="Automate the process of creating a project, setting up your development environment, and configuring automations." href="https://expo.new" Icon={Earth01Icon} /> */}
+
+<BoxLink
+  title="Quick start"
+  description="Read the docs to create a project and set up your development environment."
+  href="/get-started/create-a-project"
```

Comment:
```suggestion
  href="/get-started/create-a-project/"
```

Reply:
Oh! I didnt know we needed trailing slashes